CI/flatpak/com.obsproject.Studio.json

```diff
@@ -263,6 +263,33 @@
         }
       ]
     },
+    {
+      "name": "pipewire",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Daudiotestsrc=disabled",
+        "-Djack=disabled",
```

Comment:
At least JACK support must be enabled for your Flatpak - we do enable the JACK plugin in OBS Studio